pkg/conference/publisher/publisher.go

```diff
@@ -54,7 +54,7 @@ func NewPublisher(
 				return
 			default:
 				if err := publisher.forwardPacket(); err != nil {
-					log.Errorf("track ended: %s", err)
+					log.Infof("track ended: %s", err)
```

Comment:
This is not an error anymore?

Reply:
I've noticed that this error has been logged all the time when the tracks stopped gracefully, i.e. with the EOF, so it was not actually an error, but a normal lifecycle of a track.

But now, after your comment, I realize that I probably must not treat **each error** returned from the `forwardPacket()` as "graceful close", instead I must check if it was EOF (normal close) or if something abnormal happened and log them differently. I'll address it right away, thanks for pointing it out!